docs/.vitepress/config/sidebar.ts

```diff
@@ -27,8 +27,11 @@ export const sidebar: DefaultTheme.Sidebar = {
       collapsible: true,
       items:[
         { text: '利用软件镜像源更快地构建环境',link:'/zh/tips/mirror' },
+        
         { text: '用户相关配置', link:'/zh/tips/config'},
-        { text: 'CI/CD 集成', link:'/zh/tips/ci'}
+        { text: 'Remote Cache', link: '/tips/cache'},
+        { text: 'CI/CD 集成', link:'/zh/tips/ci'},
+        { text: '多目标构建', link:'/tips/multi-target'}
```

Comment:
No such file? For cache and multi-target.

Reply:
Yep we should use multitarget here.